backend/routes/form_routes/form_translations_routes.py

```
@@ -0,0 +1,225 @@
+        
+        # Determine target languages
+        target_languages = body.dest_languages or form.supported_locales or None
+        
+        # Request translations from the translator service
+        try:
+            translation_result = BulkTranslator.translateFromOtherMulti(
+                translatable_texts,
+                src="en",
+                dest_languages=target_languages or []
+            )
+        except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)